package.json

```diff
@@ -22,7 +22,8 @@
         "local-registry": "./scripts/local-registry.sh",
         "e2e:nextjs-app-router:start": "nx build nextjs && ./e2e/nextjs/app-router/app/start.sh",
         "e2e:nextjs-pages-router:start": "nx build nextjs && ./e2e/nextjs/pages-router/app/start.sh",
-        "e2e:js-cdn:start": "nx run js-cdn:build-with-sdk && nx run js-cdn:serve"
+        "e2e:js-cdn:start": "nx run js-cdn:build-with-sdk && nx run js-cdn:serve",
+        "e2e:js-esm:start": "nx build js && ./e2e/js/js-esm/start.sh"
```

Comment:
it would be good if the e2e app could also be type checked to make sure the sdk's type definitions are working

Reply:
what would that look like

Reply:
done